spec/Section 6 -- Execution.md

```diff
@@ -789,8 +845,8 @@ field.
 
 Since `Non-Null` type fields cannot be {null}, field errors are propagated to be
 handled by the parent field. If the parent field may be {null} then it resolves
-to {null}, otherwise if it is a `Non-Null` type, the field error is further
-propagated to its parent field.
+to {null}, otherwise if its {ModifiedOutputType} is a `Non-Null` type, the field
+error is further propagated to its parent field.
```

Comment:
If folks are cool with it, I think we should be more precise here and say that nulls are propagated rather than errors. I think the language about error propagation likely stems from the GraphQL-JS implementation which centers its logic for propagation decisions around errors, but the errors themselves technically have no impact on any fields beyond causing the originating field to become null, which may in turn violate the spec and cause propagation. The only way parent fields can "handle" an error propagated from a child field is by being `Nullable`. 

If the intent is to provide fields other means of error handling in a future version of the spec, then I could see a reason to keep the current language.